R/create_dummy_package2.R

```diff
@@ -0,0 +1,110 @@
+#' Create dummy package
+#'
+#' Creates a fuller package than \code{\link[arcticdatautils]{create_dummy_package}}
+#' but is otherwise based on the same concept. This dummy
+#' package includes multiple data objects (2 csv's, and 2 jpg's),
+#' responsible parties, geographic locations, method steps, etc.
+#'
+#' @param mn (MNode) The member node to make the dummy package on
+#' @param title (character) Optional. Title of package. Defaults to
+#' "A Dummy Package".
+#'
+#' @import arcticdatautils
+#' @import EML
+#' @import dataone
+#'
+#' @export
+#'
+#' @author Irene Steves
+
+create_dummy_package2 <- function(mn,
+                                  title = "A Dummy Package") {
+
+    stopifnot(is(mn, "MNode"))
+    stopifnot(is.character(title))
+
+    #upload objects
+    file.create(c("dummy1.csv", "dummy2.csv", "dummy1.jpg", "dummy2.jpg"))
```

Comment:
Would it be worth writing small data frames to temporary csvs?  That way each csv would have some data - I'm not sure if that adds any value to the dummy package though.

Reply:
adding that to todo :)